controllers/ControllerComment.php

```diff
@@ -81,7 +77,7 @@ public function checkComment(string $comment, int $commentId, int $postId){
 		$update = $this->_commentManager->updateComment($content, $commentId);
 
 		if ($update){
-			header('location: http://'.$this->_server.'/Post/'.$postId);
+			header('location: http://'.$this->_server.'/Post/viewPost/'.$postId);
```

Comment:
Codacy found an issue: [The use of function header() is discouraged](https://app.codacy.com/manual/PatOpen/OC_Blog/pullRequest?prid=6061935)